etc/tooling/fle.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+USAGE="usage: fle.sh <command>
+
+commands:
+	link [--build-libmongocrypt]: builds the node bindings, optionally building libmongocrypt and links it to the driver
+	rebuild: rebuilds the node bindings
+	unlink: unlinks the node bindings from the driver
+
+required environment variables:
+	node_bindings - a path, relative or absolute, to the directory that contains the Node libmongocrypt bindings"
```

Comment:
One more improvement, if y'all agree, maybe `FLE_BINDINGS` / `CSFLE_BINDINGS`? I'm thinking to make it specific if we dupe a script like this for kerberos work or other bindings down the line.

Something we could always change later too

Reply:
@nbbeeken Just to clarify, the change you're proposing here is just to change the name of the env variable with the logic being that we don't want to have any future scripts like this to be incompatible with each other, correct?

If so, then I agree it would be a quick change to make now.